### DISCUSSION TITLE
Does Longhorn support using multiple disk devices?

### DISCUSSION THREAD

**Discussion by franklin-gaoxy** [longhorn/longhorn#9255] [General]
For example, I have three separately mounted disk devices: /mnt/data01 /mnt/data02 /mnt/data03 on the host. Does Longhorn support using these three disk devices for data storage at the same time?

**Comment by mantissahz** [comment]
Here is the document for multiple disk support https://longhorn.io/docs/1.6.2/nodes-and-volumes/nodes/multidisk/

**Comment by Szwendacz99** [comment]
I think mine question #9359 is somewhat related, but my usecase is more specific (small fast & big slow storage). The answer above is almost what I need, but the disk would need to be choosable by pvc spec
